amazon-chime-sdk/src/main/java/com/amazonaws/services/chime/sdk/meetings/internal/audio/DefaultAudioClientController.kt

```diff
@@ -148,21 +162,27 @@ class DefaultAudioClientController(
         val appInfo = AppInfoUtil.initializeAudioClientAppInfo(context)
 
         uiScope.launch {
+            muteMicAndSpeaker = false
```

Comment:
`muteMicAndSpeaker` is initialized to false, and there is no place to set it to true. is this statement needed?

Reply:
+1

Reply:
Removed